src/debug.cc

```diff
@@ -27,6 +27,7 @@ bool Debug::log_syslog = false;
 int Debug::Levels[MAX_DEBUG_SECTIONS];
 char *Debug::cache_log = NULL;
 int Debug::rotateNumber = -1;
+int Debug::DroppedEarlyMessages;
```

Comment:
Please initialize explicitly (here or in the header, not sure what would be best).